testutil-server/src/test/java/io/spine/testing/client/grpc/TestClientTest.java

```diff
@@ -67,18 +65,17 @@
 
     @BeforeEach
     void setUpAll() throws IOException {
-        BoundedContextBuilder context = BoundedContext
+        var context = BoundedContext
                 .singleTenant("Tennis")
```

Comment:
Please consider putting `singleTenant()` call on the previous line.

Reply:
Should we add those methods to the IDEA's list of builder methods?